### PR TITLE
Problem: versiondb don't support restore from local snapshot

### PR DESCRIPTION
**Pull request by yihuang**
to setup a pruned versiondb node, we need to restore versiondb from local snapshot.

Solution:
- implement a new subcommand for that.

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -40,6 +40,7 @@
 - [#1083](https://github.com/crypto-org-chain/cronos/pull/1083) memiavl support both sdk 46 and 47 root hash rules.
 - [#1091](https://github.com/crypto-org-chain/cronos/pull/1091) memiavl support rollback.
 - [#1100](https://github.com/crypto-org-chain/cronos/pull/1100) memiavl support read-only mode, and grab exclusive lock for write mode.
+- [#]() versiondb support restore from local snapshot.
```
Comment (yihuang) [inline CHANGELOG.md:43]:
```suggestion
- [#1108](https://github.com/crypto-org-chain/cronos/pull/1108) versiondb support restore from local snapshot.
```

File: versiondb/client/restore.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+// RestoreVersionDBCmd returns a command to restore a versiondb from local snapshot
+func RestoreVersionDBCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "restore-versiondb <height> <format>",
+		Short: "Restore initial versiondb from local snapshot",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := server.GetServerContextFromCmd(cmd)
+
+			height, err := strconv.ParseUint(args[0], 10, 63)
```
Comment (mmsqe) [inline versiondb/client/restore.go:31]:
```suggestion
			height, err := strconv.ParseUint(args[0], 10, 64)
```

**Review by mmsqe — COMMENTED** [review]
we can remove tempfile now

File: versiondb/client/restore.go (hunk 2 above)
Comment (yihuang) [reply versiondb/client/restore.go:31]: Because it's to be converted to signed int64, so I choose 63bits here

**Review by mmsqe — APPROVED** [review]

**Pull request merged**